package.json

```diff
@@ -15,7 +15,7 @@
   "dependencies": {
     "@babel/compat-data": "^7.11.0",
     "@bcrs-shared-components/action-chip": "1.0.2",
-    "@bcrs-shared-components/certify": "1.0.15",
+    "@bcrs-shared-components/certify": "1.0.17",
```

Comment:
I'm not sure how you able to `npm i` this change since the new version is not published to npm, right? You may have to do this again after Lerna creates the tag and NPM publishes this package.

Did you follow the bcrs-shared-components steps in order?

Reply:
I made a mistake. I published from my branch instead of master 😞

Reply:
No problem. After your "shared" PR is merged, please continue with step 14 and it should sort itself out (but you might have to update the version again).